### PR TITLE
Automatically select db on new connection creation in pool

### PR DESCRIPTION
**Pull request by marrrvin**

### CODE REVIEW

**Comment by thefab** [comment]
great ! just a little change proposal

**Comment by marrrvin** [comment]
Ok. What should I do?

File: tornadis/pool.py [hunk 1]
```diff
@@ -88,6 +91,10 @@ def get_connected_client(self):
                 LOG.warning("can't connect to %s", client.title)
                 raise tornado.gen.Return(ClientError("can't connect to %s" %
                                                      client.title))
+
+            if self.db != 0:
+                yield client.call('SELECT', self.db)
```
Comment (thefab) [inline tornadis/pool.py]: can you add a test about the result of the SELECT command and raise a tornado.gen.Return(ClientError("something")) if there is a problem here ?

**Review by thefab — CHANGES_REQUESTED** [review]

File: tornadis/pool.py (hunk 1 above)
Comment (marrrvin) [reply tornadis/pool.py]: > raise a tornado.gen.Return(ClientError("something"))
> Done.
> 
> can you add a test about the result
> I have no idea how to figure out what db is selected right now (

Comment (marrrvin) [reply tornadis/pool.py]: One more thing.
Please make release after this feature will be merged.

**Review by thefab — APPROVED** [review]

**Comment by thefab** [comment]
It seems good for me, I merge it

**Comment by marrrvin** [comment]
Thank you.
Can you make new release, please?

**Comment by thefab** [comment]
I just moved your db selection code from pool to client object.

Same behaviour but you don't have db property anymore on the pool object (but on the client object)

Can you test it ?

**Comment by marrrvin** [comment]
Thanks. It works for me.

**Pull request merged**